### PR TITLE
React 17 support

### PR DESCRIPTION
**Pull request by InventingWithMonster**

### CODE REVIEW

File: src/events/__tests__/use-event.test.tsx [hunk 1]
```diff
@@ -53,13 +53,15 @@ describe("useDomEvent", () => {
...
                 const ref = useRef(document.body)
                 useDomEvent(ref, "mousedown", handler)
-                useEffect(resolve)
                 return <div />
             }
             const { rerender, unmount } = render(<Component />)
             rerender(<Component />)
             unmount()
-            fireEvent.mouseDown(document.body)
+            requestAnimationFrame(() => {
```
Comment (InventingWithMonster) [inline src/events/__tests__/use-event.test.tsx:61]: I think 17's unmounting logic is asynchronous

File: src/gestures/__tests__/tap.test.tsx [hunk 2]
```diff
@@ -70,35 +70,6 @@ describe("tap", () => {
 
         expect(tap).toBeCalledTimes(1)
     })
-
-    test("tap event listeners are cleaned up when unmounted", () => {
```
Comment (InventingWithMonster) [inline src/gestures/__tests__/tap.test.tsx:74]: This is essentially the same test as the fixed use-event test

File: src/motion/__tests__/component.test.tsx [hunk 3]
```diff
@@ -243,27 +243,6 @@ describe("motion component rendering and styles", () => {
...
-    it.skip("style supports originX and originY", () => {
-        const { container } = render(<motion.div style={{ originX: "25%" }} />)
-        expect(container.firstChild).toHaveStyle("transform-origin: 25% 50%")
-    })
-
-    it("updates style for transform values", () => {
-        let rotate = 0
-
-        const Component = () => {
-            rotate += 45
```
Comment (InventingWithMonster) [inline src/motion/__tests__/component.test.tsx:257]: This was an unsafe mutation and this use-case is covered in the build-html-styles tests

**Comment by joshbedo** [comment]
This is merged right? I can't install framer motion with React 17.0.1. Getting this error.

**Pull request merged**